examples/minimal/packages/client-react-external-wallet/src/mud/setupNetwork.ts

```diff
@@ -18,27 +30,28 @@ export async function setupNetwork() {
 
   const publicClient = createPublicClient(clientOptions);
 
-  const { tables, useStore, latestBlock$, storedBlockLogs$, waitForTransaction } = await syncToZustand({
+  const syncResult = await syncToZustand({
     config: mudConfig,
     address: networkConfig.worldAddress,
     publicClient,
     startBlock: BigInt(networkConfig.initialBlockNumber),
   });
 
   const write$ = new Subject<ContractWrite>();
-  let nextWriteId = 0;
-  const onWrite = (write: ContractWrite) =>
-    write$.next({ id: `${write.id}:${nextWriteId++}`, request: write.request, result: write.result });
+
+  const burnerClient = createWalletClient({
+    ...clientOptions,
+    account: privateKeyToAccount(getBurnerPrivateKey("mud:example:burnerWallet")),
+  })
+    .extend(publicActions)
+    .extend(burnerActions)
+    .extend(setupObserverActions(write$));
```

Comment:
write$ and the burnerClient now look like this.

In this burner wallet setup, it's natural to have write$ here. If we don't have a wallet client at this setup phase, we can create write$ when getting/creating the wallet client, and it should be kept as long as the same wallet client is in use.

I thought it's nice to update dev-tools like these, although they may not be immediately necessary:

- Allow changing the wallet client, or make it easier to unmount it.
- Make the wallet client optional. Even without it, it's still really useful.